DOC-2270: add additions documentation for TINY-9979 to the release notes (#3156)
tinymce/tinymce-docs: modules/ROOT/pages/7.0-release-notes.adoc

// CCFR here.

=== Element preset support for the `valid_children` option and `Schema.addValidChildren` API.

{productname} 7.0 updates the xref:content-filtering.adoc#valid_elements[`+valid_elements+`] option and `schema.addValidChildren` function, which now support preset names such as `@blocks`, `@phrasing`, and `@flow`. These presets will be automatically expanded into corresponding lists of valid elements, simplifying the definition process.

> For instance, specifying valid_children: `+'custom-elm[@flow]'+` will configure the custom-elm element to contain all elements considered as flow elements according to the schema. This streamlines the process of defining valid elements and eliminates the need to manually list all elements from the schema spec.

Reviewer: ```suggestion
For instance, specifying valid_children: `+'custom-elm[@flow]'+` will configure the `custom-elm` element to allow containing all elements considered as flow elements according to the schema. This streamlines the process of defining valid elements and eliminates the need to manually list all elements from the schema spec.
```

> For instance, specifying valid_children: `+'custom-elm[@flow]'+` will configure the `custom-elm` element to allow containing all elements considered as flow elements according to the schema. This streamlines the process of defining valid elements and eliminates the need to manually list all elements from the schema spec.

Reviewer 2: ```suggestion
For instance, specifying valid_children: `+'custom-elm[@flow]'+` will configure the `custom-elm` element to accept any child element classified as a flow element. This streamlines the process of defining valid elements and eliminates the need to manually list all elements from the schema spec.
```

Maybe? 😬
